README.md

```diff
@@ -1,57 +1,87 @@
 # mindlogger-report-server
+
 Server for MindLogger PDF reports
 
 ## Requirements
-- Node >= 16
+
+- Node >= 20
 
 ## Installation
 
 ### Install dependencies
+
 ```
 npm install
 ```
+
```

Comment:
Not sure how many are affected by this, but could you add the helpful tip you provided about MacOSX 14.5

> FYI:  if anyone updated your macos to latest version and having error in "npm install" for report server, the solution is
`export CPPFLAGS="-I/Library/Developer/CommandLineTools/SDKs/MacOSX14.5.sdk/usr/include/c++/v1/"`
before npm install

Reply:
This PR will update to node 20 and also update very old dependencies, include the one which fail in new macos, so this PR also fix this issue